src/api/CoinBase.ts

```diff
@@ -27,33 +28,28 @@ const CURRENCY_MAPPING: Record<string, string> = {
 
 export default class CoinBase extends BaseExchangeApi {
     constructor() {
-        super('CoinBase', CURRENCY_MAPPING, ['RIF']);
+        super('CoinBase', CURRENCY_MAPPING, ['RIF', 'RBTC']);
     }
 
     async queryExchangeRate(
         sourceCurrency: string,
         targetCurrency: string
     ): Promise<BigNumber> {
         const upperCaseTargetCurrency = targetCurrency.toUpperCase();
-        const response: Response = await fetch(
-            `${URL}?currency=${sourceCurrency}`,
-            {
-                method: 'get'
-            }
-        );
+        let response: AxiosResponse;
 
-        if (!response.ok) {
-            const {
-                errors: [error]
-            }: CoinBaseErrorResponse = await response.json();
+        try {
+            response = await axios.get(`${URL}?currency=${sourceCurrency}`);
+        } catch (error: unknown) {
+            const { response } = error as AxiosError;
             throw Error(
-                `CoinBase API status ${response.status}/${response.statusText}/${error.message}`
+                `CoinBase API status ${response.status}/${response.statusText}`
             );
         }
 
         const {
             data: { rates }
-        }: CoinBaseResponse = await response.json();
+        }: CoinBaseResponse = await response.data;
```

Comment:
Hm so the object goes: response.data.data?

Also remove await.

Reply:
Done